### ISSUE TITLE
'neon' cannot be used as a value because it was exported using 'export type'

### ISSUE THREAD

**Issue by russelldias98** [neondatabase/serverless#56]
## Steps to reproduce
```
import { neon } from '@neondatabase/serverless';
import { drizzle } from 'drizzle-orm/neon-http';

const sql = neon(process.env.DATABASE_URL!);
const db = drizzle(sql);
```

## Expected result
No errors and neon function to work

## Actual result
```
'neon' cannot be used as a value because it was exported using 'export type'.ts(1362)
index.d.mts(1, 1): 'neon' was exported here.
```

## Environment
- Node v18.18.2
- NextJS v14.0.4

**Comment by jawj** [comment]
I've just had a similar report offline. Must be related to yesterday's release of 0.7. This is the first thing on my list today.

**Comment by jawj** [comment]
Should be fixed in 0.7.1. Do let me know if not.

**Comment by russelldias98** [comment]
@jawj works like a charm!
